Source/Scene/Scene.js

```diff
@@ -945,6 +950,40 @@ define([
         },
 
         /**
+         * This property is for debugging only; it is not for production use.
+         * <p>
+         * When <code>true</code>, draws primitives to show the boundaries of the camera frustum
+         * </p>
+         *
+         * @type Boolean
+         *
+         * @default false
+         */
+        debugShowFrustumPlanes : {
+            get : function() {
+                return this._debugShowFrustumPlanes;
+            },
+
+            set : function(val) {
+                if (!val) {
+                    if (defined(this._debugFrustumPlanes)) {
+                        this.primitives.remove(this._debugFrustumPlanes);
+                        this._debugFrustumPlanes = this._debugFrustumPlanes && !this._debugFrustumPlanes.isDestroyed() && this._debugFrustumPlanes.destroy();
+                    }
+                } else if (val !== this._debugShowFrustumPlanes) {
+                    this._debugFrustumPlanes = this._debugFrustumPlanes && !this._debugFrustumPlanes.isDestroyed() && this._debugFrustumPlanes.destroy();
+                    this._debugFrustumPlanes = new DebugCameraPrimitive({
+                        camera: this.camera,
+                        updateOnChange: false
+                    });
+                    this._debugFrustumPlanes.update(this.frameState);
+                    this.primitives.add(this._debugFrustumPlanes);
```

Comment:
Do not add directly to `primitives`. Instead `Scene` should be responsible for pushing `this._debugFrustumPlanes`'s commands.